vlib/x/templating/dtm/README.md

```diff
@@ -0,0 +1,193 @@
+# dtm - Dynamic Template Manager
+
+A simple template manager integrated into the V project, designed to combine the power of V
+templates with Vweb, without the need to recompile the application with every change.
+
+**_This module is in the experimental phase, it lacks some features and may contain bugs_**
+
+## Quick Start
+
+Using the dynamic template manager ( named '**dtm**' in this readme) is relatively straightforward.
+You just need to create an instance and initialize it. Then, call the '**serve_dynamic_template**'
```

Comment:
Is there a use case, where you may want to create an instance, and use it without initializing it?
If no, merging the initialization into the creation can simplify uses.

Reply:
Would not just `expand` be clearer as a name? The instance on which you call it, is already called `dtm`, there is little point in repeating the `dynamic_template` in the method name too, and `serve` is too generic imho.